src/views/Employees.vue

```diff
@@ -38,7 +38,7 @@
 
 <script>
 import ToolbarForm from "../components/ToolbarForm";
-import * as APIService from "../APIService";
+import * as employees from "../api/employees";
```

Comment:
It would be better to get all methods from `employees` api file as the named imports.

Reply:
fixed